tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -90,168 +69,148 @@ This alternative approach requires that you have npm installed on your machine.
     }
     ```
 
-3. Install AppRouter dependencies using the following commands
-
+3. Within `<destLocation>/approuter` create a new file called `xs-app.json` with the following content:
+    ```json
+    {
+      "welcomeFile": "index.html",
+      "routes": [{
+        "source": "/",
+        "target": "/",
+        "destination": "app-destination"
+        }]
+    }
     ```
-    npm config set @sap:registry https://npm.sap.com
-    npm install
+
+4. Now head to `<destLocation>/firstapp` and replace the `random-route: true` in `manifest.yml` with the following content:
+
+    ```yaml
+    routes:
+      - route: firstapp-<SUBACCOUNT>.cfapps.<REGION_IDENTIFIER>.hana.ondemand.com
     ```
 
-[DONE]
-[ACCORDION-END]
+    Replace `<SUBACCOUNT>` with your actual account, which is the one you use in the CF CLI to login to Cloud Foundry. With this step we determine a route instead of generating a random route. This is important since we will configure the app router for exactly this destination. Further change the `<REGION_IDENTIFIER>` to match your region (e.g. `eu10`). If you are unsure about your region, head to your Cloud Foundry Cockpit Home Screen. More details on the region specific URLs can be found [here](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/350356d1dc314d3199dca15bd2ab9b0e.html).
```

Comment:
See above, move this down, but add an explanation on how to identify the value for `<SUBACCOUNT>`. This is not self-evident for readers. They need to go into the SCP cockpit and look for the subdomain associated with the subaccount, so a screenshot will be helpful.

Reply:
AFAIK the subaccount is the one required when logging into CF with `cf login`. Given the instructions in [creating the sample app](https://developers.sap.com/tutorials/s4sdk-cloud-foundry-sample-application.html) I feel it should be clear. Or am I missing something here?